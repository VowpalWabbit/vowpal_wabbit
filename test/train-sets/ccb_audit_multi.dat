ccb shared | a
ccb action | b
ccb action |
ccb slot |s a
ccb slot |v b

