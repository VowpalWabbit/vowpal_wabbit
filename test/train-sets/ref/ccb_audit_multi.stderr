creating quadratic features for pairs: ::
using no cache
Reading datafile = train-sets/ccb_audit_multi.dat
num sources = 1
Num weight bits = 18
learning rate = 0.5
initial_t = 0
power_t = 0.5
Enabled reductions: gd, generate_interactions, scorer-identity, csoaa_ldf-rank, cb_adf, cb_explore_adf_greedy, cb_sample, shared_feature_merger, ccb_explore_adf
Input label = ccb
Output pred = decision_probs
average  since         example        example  current  current  current
loss     last          counter         weight    label  predict features
    n.a.     n.a.            1            1.0      ?,?      0,1       41

finished run
number of examples = 1
weighted example sum = 1.000000
weighted label sum = 0.000000
average loss = n.a.
total feature number = 41
